styles/style.css

```diff
@@ -0,0 +1,219 @@
+* {
+  box-sizing: border-box;
+}
+
+.header-img {
+  background-image: url(../images/header-image.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 20em;
+}
+
+.logo {
+  max-width: 40%;
+  margin-left: 1em;
+  margin-top: 1em;
+}
+
+.tagline {
```

Comment:
watch your css formatting to make sure things are aligned right.
```
.tagline {
  color: black;
}
```

Reply:
I'm not sure what formatting you mean. I removed an extra space on line 19.